route/build/src/lake_route.f90

```diff
@@ -150,6 +172,14 @@ subroutine lake_route(&
         case (1)
           ! the model is Doll03
           ! print*, "lake model is Doll 2003"
+
+          ! temporary solution, this should be removed if there is restart activated...
+          print*, iTime, 'iTime in the area'
+          if (iTIme == 1) then
+            RCHFLX_out(iens,segIndex)%REACH_VOL(1) = RPARAM_in(segIndex)%D03_MaxStorage
+          endif
+
+          !
```

Comment:
I think you can remove this

Reply:
@nmizukami this is added together with the `is_vol_wm_jumpstart` flag which easier scenario building with the user. in this case the user needs to changed the target volume and if this flag is on the volume for the lakes and reservoir following a target volume will be exactly the same value as in the target volumes time series. this is however different from initializing the parametric lakes that can be done via restart file. any thoughts suggestion on that?

Reply:
sorry I confused this with the flag to jump start the state of lake, this is commented and will be removed.